package.json

```diff
@@ -40,6 +42,10 @@
     "@babel/preset-env": "^7.7.6",
     "@testing-library/dom": "^7.5.1",
     "@testing-library/preact": "^1.0.2",
+    "@types/node": "^14.0.1",
+    "@types/react": "^16.9.35",
+    "@types/react-redux": "^7.1.8",
```

Comment:
Are we using `react-redux` anywhere?

Reply:
We introduced it for some tests, but I have removed those references, so I can get rid of these extra deps for now